src/content/docs/agile-handbook/sprint-mechanics/liaisonships.mdx

```diff
@@ -230,8 +230,4 @@ Not every project gets a liaisonship! For smaller projects, we'll encourage team
   >
     What is a hero? →
   </ButtonLink>
-</ButtonGroup>
-
-<Callout variant="tip">
-  We welcome thoughts or questions on our handbook! The easiest way to get in touch is to [file a GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
-</Callout>
+</ButtonGroup>
```

Comment:
The liaisonship doc doesn't quite seem to line up with the current organizational model @austin-schaefer. But maybe that can wait until another time?